### ISSUE TITLE
Publishing/release automation

### ISSUE THREAD

**Issue by aulneau** [stx-labs/stacks.js#1057]
For our other projects, we have some process for automating releases. Looking at `stacks.js`, there isn't any automation currently (eg, merge to main -> production deployment to npm).

Does splat have interest in this, or is it there a need to have the manual publishing step?

Additionally, would it be possible to publish new versions of all packages that have changes? There are many changes on `master` that have yet to be published. cc @reedrosenbluth @zone117x

**Comment by zone117x** [comment]
@reedrosenbluth @CharlieC3 did either of you have a chance to discuss this? 
I think steps here would be: 
1. document current manual release process
2. @CharlieC3 helps out with gh actionss to automate

Thoughts?

**Comment by CharlieC3** [comment]
@zone117x We had a meeting about this on Friday. We ended up deciding that @reedrosenbluth would start work on the release automation for `stacks.js` following the example laid out in [this repo](https://github.com/hirosystems/gh-actions-example), and [with this extension](https://github.com/pmowrer/semantic-release-monorepo). I'm happy to provide assistance as well.

That being said, I just found an [issue in the DevOps repo](https://github.com/hirosystems/devops/issues/726) I created back in June to work on the release process for this repo. Should I close this ticket?
